config/v1/types_infrastructure.go

```diff
@@ -1102,6 +1182,8 @@ type VSpherePlatformNodeNetworking struct {
 // VSpherePlatformSpec holds the desired state of the vSphere infrastructure provider.
 // In the future the cloud provider operator, storage operator and machine operator will
 // use these fields for configuration.
+// +kubebuilder:validation:XValidation:rule="!has(oldSelf.apiServerInternalIPs) || has(self.apiServerInternalIPs)", message="Value is required once set"
+// +kubebuilder:validation:XValidation:rule="!has(oldSelf.ingressIPs) || has(self.ingressIPs)", message="Value is required once set"
```

Comment:
Please update the error messages here to explain which field you are referring to, the error coming out won't tell the user which field failed because you're adding this to the struct

Reply:
Fair enough, fixed